src/main/java/io/cdap/e2e/pages/actions/CdfSysAdminActions.java

```diff
@@ -77,4 +84,237 @@ public static void verifySuccess() {
     }
     Assert.assertTrue(checkParam);
   }
+
+  /**
+   * Click on the System Admin Menu
+   */
+  public static void clickOnSystemAdminMenu(String menuName) {
+    ElementHelper.clickOnElement(CdfSysAdminLocators.locateMenu(menuName));
+  }
+
+  /**
+   * Select the type of preferenceName to open in system admin
+   *
+   * @param preferenceName @data-cy attribute value of preference name. If preferenceName is present
+   *                       in {@link ConstantsUtil#DEFAULT_DATACY_ATTRIBUTES_FILE} then its data-cy
+   *                       is fetched from it else preferenceName is used as it is.
+   */
+  public static void clickOnHeading(String preferenceName) {
```

Comment:
Can we change the method name to something more specific.

Reply:
Change done.